Tools/dollymix/zslice.py

```
@@ -0,0 +1,65 @@
+
+for i in range(amount):
+    name = basefile+str(i+1)
+    filename = name+ext
+    states.append(name)
+    print(filename)
+    if(reverse):
+        base.crop((0,size*(amount-1-i),base.width,size*(amount-i))).save(os.path.concat(path,filename))
+    else:
+        base.crop((0,size*i,base.width,size*(i+1))).save(os.path.concat(path,filename))
+
+unshadedfile = os.path.concat(path,basefile+"-unshaded"+ext)
```

Pylint (3.3.7)

[error] Module 'posixpath' has no 'concat' member

(E1101)